src/lib.rs

```diff
@@ -1841,16 +1841,16 @@ pub unsafe trait TryFromBytes {
     /// # use zerocopy_derive::*;
     ///
     /// // The only valid value of this type is the byte `0xC0`
-    /// #[derive(TryFromBytes, KnownLayout, Immutable)]
```

Comment:
Don't need the `Immutable` bound either, right?

Reply:
Done